### PR TITLE
Add dracula.js

### PR DESCRIPTION
**Pull request by strathausen**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/strathausen/dracula
 * Official website (optional, not the repository): https://www.graphdracula.net
 * NPM package url (optional): https://www.npmjs.com/package/graphdracula
 * GitHub / Bitbucket popularity (required):
   - Count of stars:
   - Count of watchers:
   - Count of forks:
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

a graph drawing library in JavaScript, see www.graphdracula.net

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@strathausen would you please fix the errors? `tools/fixFormat.js` will help you, thanks.

**Comment by PeterDaveHello** [comment]
@cdnjs/intern2 please help review this PR.

**Comment by strathausen** [comment]
Hey, thanks for taking the time! I'm a bit confused over whether `filename` should exist or not.

File: ajax/libs/graphdracula/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/strathausen/dracula.git"
+  },
+  "dependencies": {
+    "lodash": "^4.14.1",
+    "uuid": "^2.0.2"
+  },
+  "optionalDependencies": {},
```
Comment (pvnr0082t) [inline ajax/libs/graphdracula/package.json]: @strathausen Let's remove empty field.

File: ajax/libs/graphdracula/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "autoupdate": {
```
Comment (pvnr0082t) [inline ajax/libs/graphdracula/package.json]: We prefer to use npm auto-update config if the library had been published on npm. Thanks. ref: [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md)

**Comment by pvnr0082t** [comment]
@strathausen You can commit v1.0.2 files under `dist` folder and put them into `v1.0.2` folder, or remove the version info in package.json file.

**Comment by PeterDaveHello** [comment]
@strathausen Should exist.

File: ajax/libs/graphdracula/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "author": {
+    "name": "Johann Philipp Strathausen",
+    "email": "philipp@stratha.us",
+    "url": "http://stratha.us"
+  },
+  "filename": "dracula.min.js",
+  "name": "graphdracula",
+  "description": "JavaScript Graph Layouting and Drawing",
+  "homepage": "http://www.graphdracula.net",
```
Comment (kennynaoh) [inline ajax/libs/graphdracula/package.json]: It's better to use `https://www.graphdracula.net/`
Thanks!

**Comment by strathausen** [comment]
Thanks, good catch! I'll also update the homepage on npm.

**Comment by strathausen** [comment]
Is it good to go now, or something still missing?

**Comment by PeterDaveHello** [comment]
@pvnr0082t @kennynaoh please help review, thanks.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
@strathausen thanks for sending this PR and be patient to be reviewed by our team members.
@pvnr0082t @kennynaoh well done!

**Comment by PeterDaveHello** [comment]
Change the title to be more clean and clear.

**Comment by PeterDaveHello** [comment]
@strathausen : graphdracula v1.0.1-x files under `dist` have different structure with v1.0.2/1.0.3, seems to be not bundled and not minified, do you want them to be hosted on CDNJS? If so, is that the correct structure to be hosted? Thanks.

**Comment by strathausen** [comment]
@PeterDaveHello I don't need v1.0.1-x to be hosted at all, newer versions are enough! Thanks people!

**Pull request merged**